rpms/SPECS/4.2.0/wazuh-agent-4.2.0.spec

```diff
@@ -435,6 +443,14 @@ if [ -f %{_localstatedir}/tmp/wazuh.restart ]; then
   fi
 fi
 
+if [ -d %{_localstatedir}/logs/ossec ]; then
```

Comment:
What if we include an `and` in the if condition about `%{_localstatedir}/logs/wazuh` exist? For `%{_localstatedir}/logs/ossec/` removal the condition must be both folders exist, right? Just to be sure.

Reply:
These changes only apply from 4.2.0 onward so the new directories will always be present.